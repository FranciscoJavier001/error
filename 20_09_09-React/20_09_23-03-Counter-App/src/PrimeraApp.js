import React from 'react';
// import React, { Fragment } from 'react';

const PrimeraApp = () => {
    return (
        <>
            <h1>Hola Mundo</h1>
            <p>Mi Primera Applicacion</p>
        </>
    );
}

export default PrimeraApp;