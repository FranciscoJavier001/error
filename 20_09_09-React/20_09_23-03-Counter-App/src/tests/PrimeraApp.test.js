import React from 'react';
import '@testing-library/jest-dom';
import {shallow} from 'enzyme';

import PrimeraApp from '../PrimeraApp';

describe('Pruebas en <PrimeraApp />', () => {
    
    // test('debe de mostrar el mensaje "Hola, Soy Francisco ', () => {
    //     const saludo = 'Hola, Soy Francisco';
    //     const {getByText} = render(<PrimeraApp saludo={saludo}/>);
    //     expect(getByText(saludo)).toBeInTheDocument();
    // })
    
    test('debe de mostrar <PrimeraApp />> correctamente', () => {

        const saludo = 'Hola, Soy Francisco';
        const wrapper = shallow(<PrimeraApp saludo={saludo}/>) 

       expect(wrapper).toMatchSnapshot();
    });
})
