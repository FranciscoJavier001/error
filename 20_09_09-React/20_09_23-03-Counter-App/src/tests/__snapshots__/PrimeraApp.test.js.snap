// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <PrimeraApp /> debe de mostrar <PrimeraApp />> correctamente 1`] = `
<Fragment>
  <h1>
    Hola, Soy Francisco
    !!!
  </h1>
   
  <p>
    Soy un subtitulo
  </p>
</Fragment>
`;

exports[`Pruebas en <PrimeraApp /> debe de mostrar el subtitulo enviado por props : Soy un subtitulo 1`] = `"Soy un subtitulo"`;

exports[`Pruebas en <PrimeraApp /> debe de mostrar el subtitulo enviado por props : Soy un subtitulsadbasjhbo 1`] = `"Soy un subtitulsadbasjhbo"`;
